src/components/Steps/Steps.js

```diff
@@ -20,10 +20,10 @@ export default function Steps(props) {
 
         if (props.horizontalLabel) {
           icon = <div class="chi-steps__icon"></div>;
-          label = <a href="#">{props[`step${i}`]}</a>;
+          label = <div class="chi-steps__content"><a href="#">{props[`step${i}`]}</a></div>;
```

Comment:
Create and import the `STEP_CLASSES` and add it to all the necessary classes in this file.